static/js/chart/types.ts

```diff
@@ -89,6 +89,7 @@ export interface PageData {
   nearbyPlaces: string[];
   categories: { string: string };
   names: { string: string };
+  locale: string[];
```

Comment:
It doesn't look like this is used (and it shouldn't be necessary). Please revert this change.

Reply:
Thanks for the review. Done.